doc/ref/configuration/minion.rst

```diff
@@ -448,6 +448,20 @@ FQDN (for instance, Solaris).
 
     append_domain: foo.org
 
+.. conf_minion:: minion_id_lowercase
+
+``minion_id_lowercase``
+-----------------
```

Comment:
The dashed line underneath should be the same length as the line above, to prevent Sphinx build warnings.